src/setup/deployment/packaging/zip.go

```diff
@@ -67,20 +69,80 @@ func GetZippedBinaryFileSize(experimentID int, binaryPath string) int64 {
 	return zippedBinarySizeBytes
 }
 
-//GenerateZIP creates the zip file for deployment
-func GenerateZIP(experimentID int, fillerFileName string, binaryPath string) string {
+func GenerateFillerFile(experimentID int, fillerFilePath string, sizeBytes int64) {
```

Comment:
Culd you addd a unit test for this function which also checks if the file size is what it needs to be?

Reply:
Just added it :)